assets/dev/js/editor/document/callback/hooks/delete/section-columns.js

```diff
@@ -9,6 +9,10 @@ export class SectionsColumns extends HookAfter {
 		return 'delete-section-columns';
 	}
 
+	bindContainerType() {
+		return 'column';
+	}
+
 	getConditions( args ) {
 		const { containers = [ args.container ] } = args;
```

Comment:
1.  Why only here it's multiple containers?
2. It's needed? (always it will be a column)